gfx/video_crt_switch.h

```diff
@@ -33,6 +33,8 @@ void crt_aspect_ratio_switch(unsigned width, unsigned height);
 
 void crt_video_restore(void);
 
+static int crt_compute_dynamic_width(int width);
+
```

Comment:
This doesn't make a lot of sense to me. Static function prototypes should not be in a header file.

Reply:
Hmm this was changed. And is not static in my master

Reply:
OK.